tests/cupy_tests/random_tests/test_generator.py

```diff
@@ -1215,6 +1216,13 @@ def test_randint_float1(self):
     def test_randint_float2(self):
         self.generate(6.7, size=(2, 3))
 
+    @pytest.mark.skip(not sys.platform.startswith('win32'),
+                      reason='default int is 64 bits')
+    def test_randint_int32_1(self):
+        self.generate(2**16, 2**28, 3)
+
+    @pytest.mark.skip(sys.platform.startswith('win32'),
+                      reason='default int is 32 bits')
```

Comment:
I think we should fix this instead of skipping the test.
Let me look at it

Reply:
just verified this happens in windows too.
We should set the default type in cupy to int32 and pass a dtype='q' in this test

Reply:
Yeah it'd be the best if we could fix it. Now I think again, it's probably just a matter of changing `dtype='l'` to `'q'` here?
https://github.com/cupy/cupy/blob/d8e0bfda93fae1106bbff63d6fd053cb10f2b6c4/cupy/random/_sample.py#L87
By the way NumPy defaults to `dtype=int`, not sure what it means on Windows...

Reply:
My browser had a glitch and I just saw your reply after I posted mine 😅 Question is how to pass in a dtype here. I tried but didn't seem straightforward without refactoring the test.